### ISSUE TITLE
Edge + Narrator does not read dialog titles correctly

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor-dev#1368]
## Are you reporting a feature request or a bug?

Bug

## Provide detailed reproduction steps (if any)

1. Go to https://sdk.ckeditor.com/samples/fullpreset.html.
2. Open any dialog in  CKEditor (e.g. link dialog).

### Expected result

Dialog title is announced by Narrator.

### Actual result

Dialog title is not announced.

It happens because dialog title has `role="presentation"`, if the `role` is changed to a `heading` it is announced correctly by Narrator (however, should be checked if it is compatible with other environments, especially Firefox + JAWS).

Note: The officially supported environment is [Firefox + JAWS](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_browsers-section-accessibility-support), however if others can be improved we should definitely take a look on this issue.

## Other details

* Browser: Edge + Narrator
* OS: Windows 10
* CKEditor version: 4.8.0
* Installed CKEditor plugins: Tested on `full` preset

**Comment by mlewand** [comment]
As you mentioned this is not a supported setup. Narrator lacks support for multiple cases, and techniques we use for dialog title might be one of them. It takes substantial amount of time to work around these limitations, this is why we're limited to supporting JAWS only in a first place.
